ci/build_test_OnCommit.groovy

```diff
@@ -43,42 +43,25 @@ pipeline {
             }
           }
         }
-        stage('Style, SDL and clean') {
```

Comment:
Keep it parallel but add

agent {
                label "${agent_name_linux}"
              }